### PR TITLE
add files and tests

### PR DESCRIPTION
**Pull request by MajaC021**
New repo and clear branch

### CODE REVIEW

File: cypress/integration/boards.spec.js [hunk 1]
```diff
@@ -4,47 +4,113 @@ import dataBoard from "../fixtures/data.json"
...
         cy.get(boards.saveUpdate).eq(0).click()
+
+        cy.get('.el-message__group > p')
+            .should('be.visible')
+            .and('contain', 'Successfully updated the Board Basic Info.')
     })
 
-    it("Delete board", () => {
-        cy.get(boards.activeBoard).eq(0).click();
+    it.only("Delete board", () => {
```
Comment (bdimitrovski) [inline cypress/integration/boards.spec.js]: Please remove `only` before committing the final code, as this will only execute this specific test.

File: cypress/integration/boards.spec.js [hunk 2]
```diff
@@ -4,47 +4,113 @@ import dataBoard from "../fixtures/data.json"
...
         cy.get(loginModel.logInBtn).click();
 
+        cy.get('[class=vs-u-text--uppercase]')
+            .should('be.visible')
+            .and('contain', 'My Organizations')
+
+        cy.get('span.el-dropdown-link').should(($loggedInUser) => {
+            expect($loggedInUser).to.contain('Maja Cveticanin')
+        })
+        cy.get('.vs-c-modal__body > .el-button > .el-icon-close').click()
```
Comment (bdimitrovski) [inline cypress/integration/boards.spec.js]: What does this do? Close the modal? If so, could you please move it to POM and make a selector out of it?

File: cypress/integration/boards.spec.js [hunk 3]
```diff
@@ -4,47 +4,113 @@ import dataBoard from "../fixtures/data.json"
...
     });
 
+    afterEach("logout user", () => {
+        cy.get("span[class='vs-c-user-name']").click()
+        cy.get('[data-cy="account-profile"]').click()
+        cy.get('button[class="vs-c-btn vs-c-btn--link vs-c-btn--danger"]').click()
```
Comment (bdimitrovski) [inline cypress/integration/boards.spec.js]: This should also be in an elements object.

File: cypress/integration/boards.spec.js [hunk 4]
```diff
@@ -4,47 +4,113 @@ import dataBoard from "../fixtures/data.json"
...
+        })
+    })
+
+    it("Edit board without name", () => {
+        cy.get(boards.activeBoard).eq(3).click();
+        cy.get(boards.editBoard).eq(8).click();
+        cy.get(boards.edinBrdName).eq(0).clear()
+
+        cy.get("button[type='submit']").should('be.disabled')
+        cy.get("span[class='el-form-item__error el-form-item-error--top']").should('contain', "The board title field is required")
```
Comment (bdimitrovski) [inline cypress/integration/boards.spec.js]: Should also probably be stored in some JSON file with selectors

File: cypress/integration/boards.spec.js [hunk 5]
```diff
@@ -4,47 +4,113 @@ import dataBoard from "../fixtures/data.json"
...
+
+         //assert that  we logged out
+        cy.get('button[type="submit"]').should('be.visible')
+        cy.get('h1').should('contain', 'Log in with your existing account')
+
+    })
+
     //positive
     it("Add new board", () => {
         cy.get(boards.addNewBoardBtnDropdown).eq(1).click({ force: true });
```
Comment (bdimitrovski) [inline cypress/integration/boards.spec.js]: Why do we need to use `{ force: true }` here?

File: cypress/integration/boards.spec.js [hunk 6]
```diff
@@ -4,47 +4,113 @@ import dataBoard from "../fixtures/data.json"
...
         cy.get(boards.checkScrumRb).click();
         cy.get(boards.nextBtn).click();
         cy.get(boards.nextBtn).click();
         cy.get(boards.nextBtn).click();
         cy.get(boards.nextBtn).click();
-    })
 
-    it("Check board exist", () => {
-        cy.get("ul[class='vs-c-list vs-c-list--boards']").should('contain', dataBoard.board.brdName)
+        cy.get('li[class="vs-c-list__item has-caret"]').should(($loggedInUser) => {
```
Comment (bdimitrovski) [inline cypress/integration/boards.spec.js]: Stored in a file?

File: cypress/integration/organizations.spec.js [hunk 7]
```diff
@@ -11,24 +11,23 @@ describe("create organization", () => {
...
-    
+
     afterEach("logout user", () => {
         cy.get('span[class="vs-c-user-name"]').click()
         cy.get('li[data-cy="account-profile"]').click()
         cy.get('button[class="vs-c-btn vs-c-btn--link vs-c-btn--danger"]').click()
 
-         //assert that  we logged out
+        //assert that  we logged out
         cy.get('button[type="submit"]').should('be.visible')
```
Comment (bdimitrovski) [inline cypress/integration/organizations.spec.js]: Very generic, could this selector be somewhat better?

File: cypress/integration/organizations.spec.js [hunk 8]
```diff
@@ -73,11 +70,11 @@ describe("create organization", () => {
 
         cy.get("div[class='vs-c-my-organizations-item-wrapper vs-c-my-organizations-item-wrapper--archived']").should(($arhivedOrg) => {
             expect($arhivedOrg).to.contain("test")
-            })
+        })
     })
 
     it("Delete arhived org", () => {
-        cy.get(organizations.deleteArhivedOrg).eq(1).click({force: true})
+        cy.get(organizations.deleteArhivedOrg).eq(1).click({ force: true })
```
Comment (bdimitrovski) [inline cypress/integration/organizations.spec.js]: Why `{ force: true }`?

**Review by bdimitrovski — CHANGES_REQUESTED** [review]
It's fine, however, I am still missing some things here like: 
- some tests with selected board type (like creation), update board, testing character limits, etc, as well as tasks, e.g. tests for creating new columns, editing/moving/removing tasks (CRUD) etc
- some elements are still not in JSON objects so that they can be easily referenced wherever needed, without a fear that the selector might be incorrect
- a few times, I've seen the use of `{ force: true }` while clicking, whereas I'm fairly certain that this is not necessary at all

Please add in some more tests and fix these other issues before merging.

File: cypress/integration/boards.spec.js (hunk 4 above)
Comment (MajaC021) [reply cypress/integration/boards.spec.js]: 103 and 104 are asserted. Should need in the JSON file?

File: cypress/integration/tasks.spec.js [hunk 9]
```diff
@@ -1,66 +1,38 @@
...
-    afterEach("logout user", () => {
-        cy.contains("Maja C").click()
-        cy.get(logout.profile).click()
-        cy.get(logout.logoutBtn).click()
-
-        //assert that  we logged out
-        cy.get('button[type="submit"]').should('be.visible').and('contain', "Log In")
-        cy.get('h1').should('contain', 'Log in with your existing account')
-    })
+    // afterEach("logout user", () => {
```
Comment (bdimitrovski) [inline cypress/integration/tasks.spec.js]: Remove if not used

**Review by bdimitrovski — CHANGES_REQUESTED** [review]
You're again missing a lot of tests, like e.g. Create new Scrum board, open board, create board without title, create organization with empty name, archive/unarchive organizations, and many more. 

Please write some more tests to cover all of the CRUD functionality and some negative ones too. The POM is fine though.

**Review by bdimitrovski — APPROVED** [review]
Sorry, my bad, I now see that you have a lot more in this commit then what I initially looked at.  This is fine, you can merge it, but still, I'd like to see some more clean space between functions, and things like:

`span.el-form-item__error.el-form-item-error--top`, `.vs-c-img--avatar vs-c-img--board-avatar`  etc put into selector objects, where they should belong.
